contrib/systemd/torus.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Torus daemon
+Documentation=https://www.torus.sh/docs/
+After=network.target
+
+[Service]
+Type=simple
+User=torus
+Environment="TORUS_ROOT=/var/run/torus"
+EnvironmentFile=/etc/torus/token.environment
+ExecStart=/usr/bin/torus daemon start --foreground --no-permission-check
```

Comment:
I see -- it's constrained to the group. Does this also mean that they can access the TOKEN_ID and TOKEN_SECRET via `/proc` for the running user? Or is the idea that the user is locked down, then you just share secrets via group?